### ISSUE TITLE
User can submit join requests with empty social profiles

### ISSUE THREAD

**Issue by g4s8** [zerocracy/farm#915]
At moment many users submitted join requests with empty Stackoverflow and Github profiles (with 1 reputation on SO and 0-1 repos on GH). We can check it via public API and suggest to improve reputation before joining. SO reputation can be checked at: `https://api.stackexchange.com/2.2/users/{userid}?site=stackoverflow`

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by 0crat** [comment]
Job [#915](https://github.com/zerocracy/farm/issues/915) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by yegor256** [comment]
@g4s8 good point. Let's check the both and refuse if SO is less than 512 and if GitHub followers is less than 32. Make sense?

**Comment by yegor256** [comment]
@g4s8 moreover, let's add reputation numbers to the message we send to `Notify all`. The recipients will immediately see the numbers and decide whether the user is worth talking to or not.

**Comment by g4s8** [comment]
@yegor256 I think Github follower count is no such important as contribution activity and starred repos in profile. BTW only you have > 32 followers among people from https://www.0crat.com/team with reputation > 500 ;)

**Comment by yegor256** [comment]
@g4s8 let's start with reporting those numbers, just for information (SO reputation and GitHub followers) and see how it will help us?

**Comment by g4s8** [comment]
@yegor256 yes, I think we can start with it

**Comment by carlosmiranda** [comment]
@0crat wait for https://github.com/zerocracy/datum/issues/380

**Comment by carlosmiranda** [comment]
@yegor256 @g4s8 we should display these on the profile pages, correct?

**Comment by 0crat** [comment]
> @0crat wait for https://github.com/zerocracy/datum/issues/380 ([here](https://github.com/zerocracy/farm/issues/915#issuecomment-410138378))

@carlosmiranda The impediment for [#915](https://github.com/zerocracy/farm/issues/915) was registered successfully by @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)

**Comment by carlosmiranda** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/zerocracy/farm/issues/915#issuecomment-418618846))

@carlosmiranda The user @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda) resigned from [#915](https://github.com/zerocracy/farm/issues/915), please stop working. Reason for job resignation: Order was cancelled

**Comment by 0crat** [comment]
Tasks refusal is discouraged, see [§6](http://www.zerocracy.com/policy.html#6): -15 point(s) just awarded to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)

**Comment by yegor256** [comment]
@carlosmiranda yes, let's make it all visible

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) not enough funds available in the project, can't set budget of job [#915](https://github.com/zerocracy/farm/issues/915), see [§21](http://www.zerocracy.com/policy.html#21); @victornoel[/z](https://www.0crat.com/u/victornoel) will get no money on completion; in order to fix that, add funds to the project and assign the job again

**Comment by victornoel** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/zerocracy/farm/issues/915#issuecomment-427335949))

@victornoel This is what I know about this job in [C3NDPUA8L](https://www.0crat.com/p/C3NDPUA8L), as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#915](https://github.com/zerocracy/farm/issues/915) is [in scope](http://www.zerocracy.com/policy.html#14) for 6mon
  * The role is `DEV`
  * The job is assigned to @victornoel[/z](https://www.0crat.com/u/victornoel) for [1hr](http://www.zerocracy.com/policy.html#8)
  * There is no monetary reward attached, it's a [free](http://www.zerocracy.com/policy.html#2) job
  * Some equity will be [vested](http://www.zerocracy.com/policy.html#37) on completion
  * The job doesn't have any [impediments](http://www.zerocracy.com/policy.html#9)
  * The [budget](http://www.zerocracy.com/policy.html#4) is 30 minutes/points
  * These users are banned and won't be assigned:
    * @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda): User was resigned from the ticket
  * Job [footprint](https://www.0crat.com/footprint/C3NDPUA8L?q=%7Bjob%3A%27gh%3Azerocracy%2Ffarm%23915%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)
